src/AnalyzerPowerPack/AnalyzerPowerPack.md

```diff
@@ -0,0 +1,109 @@
+### CA1008: Enums should have zero value. ###
```

Comment:
@lgolding I know some of the rules are missing descriptions, and others have typos. I'll file issues for the ones I know about, but it might be useful to have someone do a pass after this is merged in and check for such issues. I can't commit to doing that myself.